docker/presets/models/tfs/Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN pip install --no-cache-dir -r /workspace/requirements.txt
 RUN apt-get update -y && apt-get install --no-install-recommends curl gcc libc-dev perl -y && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
+FROM dependencies AS base
```

Comment:
Nice change! do you need to include line 14? 
pip install requirements.txt will introduce more than 10GB space usage.
and requirements.txt was scanned by dependency bot.

Reply:
@robert-cronin are you still actively working on this?

Reply:
Hey @chewong, yep still on it!

Reply:
@zhuangqh I've changed it so that the base image is now the one with the 10GB pip dependencies